src/Front/PayplugOney/Requests/OneyBase.php

```diff
@@ -182,4 +184,12 @@ public function setOney($oney)
 		$this->oney = $oney;
 	}
 
+	public function addOneyJs() {
```

Comment:
do we need this payplug-oney.js?

Reply:
no we don't I am going to delete this, I used before changing the requirments on the US, will push a roll back